modules/configuring-albo-on-sts-cluster.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+// * networking/installing-albo-sts-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-albo-on-sts-cluster_{context}"]
+= Installing AWS Load Balancer Operator on Secure Token Service cluster
+
+The Cloud Credential Operator does not provision the credential request automatically in the STS cluster. You must set the credential request manually. You must set the credentials secret manually by using the `ccoctl` binary. The Cloud Credential Operator waits until the required secrets are created and available.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. The AWS Load Balancer Operator creates a credentials request in the `openshift-cloud-credential-operator` namespace. Extract and save the required the `credentialsrequest` object in a directory by running the following command:
++
+[source,terminal]
+----
+$ oc get credentialsrequest -n openshift-cloud-credential-operator  \
+    aws-load-balancer-controller-<cr-name> -o yaml > <path-to-credrequests-dir>/cr.yaml <1>
+----
+<1> The `aws-load-balancer-controller-<cr-name>` parameter specifies the credential request name created by the AWS Load Balancer Operator.
```

Comment:
So `cr-name` stands for the AWS Load Balancer Controller instance name. Maybe we can make this clear here.

Reply:
Addressed